src/aws-type-downloader/main.go

```diff
@@ -17,6 +17,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var skippedResources = map[string]struct{}{
```

Comment:
We skip these files that have inconsistencies in their schema and will use older versions until we have a better fix

Reply:
Is it possible for us to dynamically detect these cases? or is that not reasonable? I'm just thinking about how we move forward.